util/Setup/DockerComposeBuilder.cs

```diff
@@ -61,6 +62,7 @@ public TemplateModel(Context context)
             }
         }
 
+        public bool? EnableBuiltInMsSql { get; set; }
```

Comment:
This should probably default to true?

Reply:
@addisonbeck I talked this over with Justin, as he was involved in the initial design of the change.  He suggested making the new `EnableBuildInMsSql` Configuration option as not nullable, which I did in [`c866f42` (#4954)](https://github.com/bitwarden/server/pull/4954/commits/c866f42699a55c5ee0a70325491bd0cfc7ef5477).  

Since the only `TemplateModel` constructor always initializes the `EnableBuiltInMsSql` property from the `Context`, and the value in the `Context.Config` is defaulted to `true` (https://github.com/bitwarden/server/blob/c866f42699a55c5ee0a70325491bd0cfc7ef5477/util/Setup/Configuration.cs#L98-L99), we believe that this is not necessary to be initialized at this point as well.  Do you agree?